release-content/migration-guides/reflect_registration_changes.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Changes to type registration for reflection
+pull_requests: [15030, 20435, 20893]
+---
+
+Calling `.register_type` has long been a nuisance for Bevy users: both library authors and end users.
+This step was previously required in order to register reflected type information in the `TypeRegistry`.
+
+This comes with a few caveats however:
```

Comment:
Did you mean to first announce that the new autoreflect feature exists? Right now it reads as "calling .register_type sucks. Here are its caveats:", which is not what you meant